src/lib.rs

```diff
@@ -108,4 +108,4 @@ pub enum Error {
 }
 
 // BtlePlug Result type
-pub type Result<T> = result::Result<T, Error>;
+pub type Result<T, E = Error> = result::Result<T, E>;
```

Comment:
This seems like a weird change to make. If you want a result with a different error type somewhere, can't you just use `std::result::Result` there?

Reply:
Again just personal preference, I have started a [thread in URLO](https://users.rust-lang.org/t/dont-do-type-result-t-std-result-t-myerror/49421) about that topic. tl;dr Why should I be required to use `std::result::Result` just to state that other type parameters are used.

... back when I wrote the PR I was not that aware of 'keep a PR small and dedicated to a problem and keep personal preference out' or however it is expressed best (I'm not an english native speaker).

Reply:
It looks like it isn't used anymore anyway.